Sources/Swift/Integrations/SessionReplay/SentrySessionReplay.swift

```diff
@@ -93,12 +96,29 @@ class SentrySessionReplay: NSObject {
         delegate?.sessionReplayStarted(replayId: sessionReplayId)
     }
 
+    func pause() {
+        isSessionPaused = true
+        videoSegmentStart = nil
+    }
+    
+    func unpause() {
```

Comment:
`m`: Unpause is not used anywhere. I think we can remove it.

Reply:
Ow, yes.